plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -912,12 +912,27 @@ public TableStatistics getTableStatistics(ConnectorSession session, JdbcTableHan
     @Override
     public void createSchema(ConnectorSession session, String schemaName)
     {
+        try (Connection connection = connectionFactory.openConnection(session)) {
```

Comment:
Please use toRemoteSchema here

Reply:
I assume to make sure it behaves in the exact same way and to avoid issues If `toRemoteSchema` would be modified. `createTable` uses both `toRemoteSchema` and `toRemoteTable`.

Reply:
Done!

Reply:
@kokosing note that you previously asked me to test mixed case in the unit test but it's not possible now since `toRemoteSchema()` validates that the schema name is lowercase.

Reply:
> Why? the schema is supposed not to exist yet

It does not matter if it exists. We are changing the domain, from Trino name to remote name. So let's usage a dedicated method for that.

> does CREATE TABLE use toRemoteTable?

Yes. It is even using toUpperCase twice.